src/locales/default.json

```diff
@@ -67,6 +67,10 @@
   "quorumReached": "quorum reached",
   "options": "Option(s)",
   "votingPower": "Your voting power",
+  "reason": {
+    "label": "Reason",
+    "placeholder": "What is your reason?"
```

Comment:
I think this is to aggressive way to ask the user. Instead I would suggest

`Leave a comment (optional)`